src/Snak/SnakObject.php

```diff
@@ -102,7 +102,7 @@ public function equals( $target ) {
 	 * @return string
 	 */
 	public function serialize() {
-		return serialize( $this->propertyId->getNumericId() );
+		return serialize( $this->propertyId->getSerialization() );
```

Comment:
This is a bit pointless. Please replace with a straight `return $this->propertyId->getSerialization();`.